lazy-pages/common/src/lib.rs

```diff
@@ -74,7 +74,7 @@ pub struct GlobalsAccessError;
 /// Globals access trait.
 pub trait GlobalsAccessor {
     /// Returns global `name` value, if `name` is I64 global export.
-    fn get_i64(&self, name: &LimitedStr) -> Result<i64, GlobalsAccessError>;
+    fn get_i64(&mut self, name: &LimitedStr) -> Result<i64, GlobalsAccessError>;
```

Comment:
Is it intended change?

Reply:
Unfortunately yes. Globals acquiring requires mutable store with `wasmer`